app/assets/javascripts/app/components/button/button.js

```diff
@@ -27,7 +27,10 @@
   function councilRaisedButton() {
     return {
       restrict: 'E',
-      scope: {},
+      scope: {
+        disabled: '=',
+        click: '&'
+      },
```

Comment:
I was looking into the other components and I think we can open an exception for this case, but let's get a third opinion on this, @azevedo-252 can you?